test/WebJobs.Script.Tests.Integration/BlobLeaseManagerTests.cs

```diff
@@ -119,76 +130,6 @@ public class BlobLeaseManagerTests
         }
 
         [Fact]
-        public void AcquiringLease_WithServerError_LogsAndRetries()
```

Comment:
Are we logging anything in the failure path? I believe we're losing this information. Can we have a test to validate that?